docs/introduction.md

```diff
@@ -1,42 +1,43 @@
 # Introduction
 
-deck.gl was created to make "reactive" WebGL-based visualizations of
-"big data" simple, and to enable new users to quickly get impressive
-results, while gradually enabling more flexibility and power as
-users climb the learning curve.
+Deck.gl is a WebGL-powered framework that features an instancing and layering
+paradigm. It is specifically designed for composing and visualizing data sets
+at scale. With its reactive API design, deck.gl makes visual analytics of Big
+Data simple. It enables users to quickly prototype and get impresssive results,
+while gradually unlocking more flexibility and power as users climb the
+learning curve.
 
 
 ## Brief Overview
 
-The basic idea is to render a stack of visual overlays, usually (but
-not always) over maps.
+The basic idea of using deck.gl is to render a stack of [visual] overlays,
+usually (sbut not necessarily) over maps.
 
-While the concept is simple, to do this right, deck.gl handles
-a number of challenges, such as:
+While the concept may sound simple, to do this right, deck.gl handles a number of
+challenges, such as:
 
 * Handling of large data sets and performant updates
-* Event handling and picking
-* Cartographic Projections and Integration with underlying map
-* Selection of proven, well-tested Layers
-* Ability for the user to create new Layers and customize existing Layers
+* Interactive event handling such as picking
+* Cartographic Projections and integration with underlying map
+* Selection of proven, well-tested layers
+* Ability for the user to create new layers and customize existing layers
 
 
 ## Ecosystem
 
-deck.gl has been developed in parallel with a number of companion modules:
+Deck.gl has been developed in parallel with a number of companion modules:
```

Comment:
Yes, but grammar wise shall we keep the first letter capitalized if it is at the beginning of a sentence? (not 100% sure about this)